### DISCUSSION TITLE
Can't:

### DISCUSSION THREAD

**Discussion by Kerrisk199** [dimdenGD/OldTwitter#950] [Q&A]
Hello Dimden, I can't use any other extension while using Twitter old layout. Why? Any work around it please? Trying to use Superpowers for Twitter while using Twitter old layout as well. Kindly respond...

**Comment by dimdenGD** [comment]
because oldtwitter is completely different client

**Comment by Kerrisk199** [comment]
So there's no other way around it?

**Comment by dimdenGD** [comment]
no

**Reply by Kerrisk199** [reply]
Okay thanks
